docs/client.rst

```diff
@@ -389,7 +389,7 @@ The declaration of these functions can be found in ``uxr/client/profile/session/
 
 .. code-block:: c
 
-    uint16_t uxr_write_configure_participant_xml(uxrSession* session, uxrStreamId stream_id, uxrObjectId object_id, uint16_t domain, const char* xml, uint8_t mode);
+    uint16_t uxr_buffer_configure_participant_xml(uxrSession* session, uxrStreamId stream_id, uxrObjectId object_id, uint16_t domain, const char* xml, uint8_t mode);
```

Comment:
Increase the description. Talk about the buffering.

Reply:
Done!